pom.xml

```diff
@@ -72,6 +72,12 @@
             <artifactId>spring-boot-starter-web</artifactId>
             <version>${spring.boot.version}</version>
         </dependency>
+
```

Comment:
to musi byc tam gdzie daosy, jak bysmy to przeniesli do tsp-database to tam by wystarczylo